src/prefect_server/graphql/runs.py

```diff
@@ -150,6 +150,17 @@ async def resolve_delete_flow_run(
     return {"success": await api.runs.delete_flow_run(flow_run_id=input["flow_run_id"])}
 
 
+@mutation.field("delete_flow_runs")
+async def resolve_delete_flow_runs(
+    obj: Any, info: GraphQLResolveInfo, input: dict
+) -> dict:
+    return {
+        "affected_rows": await api.runs.delete_flow_runs(
```

Comment:
mirroring the Hasura `affected_rows` return to inspect how many ids were deleted